.packit.yaml

```diff
@@ -10,15 +10,17 @@ packages:
     upstream_package_name: flightctl
     downstream_package_name: flightctl
     upstream_tag_template: v{version} # remove v from the start of the version tags (vx.y.z)->x.y.z
+
 actions:
   get-current-version:
     - bash -c 'git describe --tags | sed "s/^v//; s/-/~/g"'
+
 jobs:
   - job: copr_build
     trigger: pull_request
     enable_net: True # this is necessary for go modules to download the sources
     targets:
-      - fedora-eln-aarch64
+      # - fedora-eln-aarch64
```

Comment:
If we are going to do this, let's add a note why and reference a ticket to reenable so we don't forget.  :pray: